spring-data-multi-storage-transaction-sample/docs/README.md

```diff
@@ -0,0 +1,286 @@
+# Spring Data Integration Sample with Multi-storage Transactions
+
+This tutorial describes how to create a sample Spring Boot application by using Spring Data integration with Multi-storage Transactions in ScalarDB.
+
+## Prerequisites
+
+- Java (OpenJDK 8 or higher)
+- Gradle
+- Docker, Docker Compose
+
+## Sample application
+
+### Overview
+
+This tutorial describes how to create a sample Spring Boot application for the same use case as [ScalarDB Sample](https://github.com/scalar-labs/scalardb-samples/tree/main/scalardb-sample) but by using Spring Data integration with Multi-storage Transaction in ScalarDB.
+Please note that application-specific error handling, authentication processing, etc. are omitted in the sample application since this tutorial focuses on explaining how to use Spring Data integration with Multi-storage Transaction in ScalarDB.
+For details, please see [Guide of Spring Data integration with ScalarDB](https://scalar-labs.github.io/scalardb-sql/spring-data-guide.html).
+
+![Overview](images/overview.png)
```

Comment:
Since this doc contains images, can we add the "spring-data-multi-storage-transaction-sample/images" folder to the "spring-data-multi-storage-transaction-sample/docs" folder as well?